packages/extension-api/src/extension-api.d.ts

```diff
@@ -4784,8 +4784,8 @@ declare module '@podman-desktop/api' {
 
     // Navigate to the Pods page
     export function navigateToPods(): Promise<void>;
-    // Navigate to a specific pod referenced by kind, name and engineId
-    export function navigateToPod(kind: string, name: string, engineId: string): Promise<void>;
+    // Navigate to a specific pod referenced by name and engineId
+    export function navigateToPod(name: string, engineId: string): Promise<void>;
```

Comment:
I don't know if we can just break changes like that, I feel like we should maybe introduce a new method, deprecate this one for a version, then remove it ?

AI Lab is using `navigateToPod` so it need to be updated, we might need to have some issue for that?

Here is a suggestion

```suggestion
    /**
     * Navigate to a specific pod referenced by kind, name and engineId
     * @deprecated will be removed in Podman Desktop 1.18 use {@link navigateToPodmanPod} instead
     * @param kind
     * @param name
     * @param engineId
     */
    export function navigateToPod(kind: string, name: string, engineId: string): Promise<void>;
    export function navigateToPodmanPod(name: string, engineId: string): Promise<void>;
```

Reply:
I was searching for the wrong thing :-/. I added a new commit that implements your suggestion, allowing AI Lab or other extensions time to migrate. The deprecated support will only work for Podman pods, but that's all AI Lab is using. I will open an issue to track the migration once we confirm this direction.

Reply:
That's good on my side !